### ISSUE TITLE
Source for importing legacy lab uses data?

### ISSUE THREAD

**Issue by melish** [eaudeweb/ozone#319]
quantity_laboratory_analytical_uses in Import, Export, Production etc.

Probably from table LabUses, but it only contains Production and Consumption columns

**Comment by melish** [comment]
From OzSec:
- The table LabUses is what we have been using to store information – the fields in Imports and Production tables were blank on our system.  When you said that you will be storing the information in the Imports and Production table, then we duplicated the info from LabUses table to the Imports/Production tables to make your importation of the information easier.  This is more so for the Imports table where the lab information had to be split across different countries. 
- So basically for your importation, you will ignore the LabUses table, but you may use the table to check if its information matches with the info in the two tables - Imports/Production tables.

**Comment by melish** [comment]
Issue can be closed - we are migrating the information from Imports/Production. Cross-check with LabUses can be done separately.
